src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -102,6 +120,56 @@ class PaymentInformation extends React.Component {
     }
   }
 
+  handleDirectDepositUpdateSubmit(data) {
+    this.props.savePaymentInformation(data);
+    recordEvent({
+      event: 'profile-transaction',
+      'profile-section': 'direct-depost-information',
+    });
+  }
+
+  handleEditClick(e) {
+    const gaProfileAction = 'edit-link';
+
+    // Open edit modal.
+    this.props.editModalToggled();
+
+    // Push Google Analytics event
+    switch (e.currentTarget.dataset.editTarget) {
+      case EDIT_TARGETS.bankName:
+        this.recordProfileNavEvent({
+          'profile-action': gaProfileAction,
+          'profile-section': 'bank-name',
+        });
+        break;
+      case EDIT_TARGETS.accountNumber:
+        this.recordProfileNavEvent({
+          'profile-action': gaProfileAction,
+          'profile-section': 'account-number',
+        });
+        break;
+      case EDIT_TARGETS.accountType:
+        this.recordProfileNavEvent({
+          'profile-action': gaProfileAction,
+          'profile-section': 'account-type',
+        });
+        break;
+      default:
+        break;
+    }
+  }
+
+  recordProfileNavEvent(customProps) {
```

Comment:
Since you're not using `this` in this method, I would probably pull it out of the component and make it a top level function in this file. That will save you some prop threading, which makes it quicker to understand what's happening, I think.

Reply:
Param removed.